config/config-defaults.yaml

```diff
@@ -42,6 +42,17 @@ data:
     # none is specified.
     revision-timeout-seconds: "300"  # 5 minutes
 
+    # max-revision-timeout-seconds contains the maximum number of
+    # seconds that can be used for revision-timeout-seconds.
+    # This value must be greater than or equal to revision-timeout-seconds.
+    # If omitted, the system default is used (600 seconds).
+    # If this value is provided but revision-timeout-seconds is omitted,
```

Comment:
I would prefer brevity here and remove lines 49-53. The behavior seems clear from the first 4 lines.